src/backend/core/api/viewsets.py

```diff
@@ -184,12 +178,69 @@ class UserViewSet(
     mixins.UpdateModelMixin,
     viewsets.GenericViewSet,
 ):
-    """User ViewSet"""
+    """
+    User ViewSet for all interactions with user infos and teams.
+
+    GET /api/users/&q=query
+        Return a list of all users whose email matches the query, completely or partially
```

Comment:
nit: the docstring could be a bit more descriptive, or explicit about the purpose of the view.

Reply:
isn't the purpose of the view to list users ? I'll add a little explanation as to the way we match the email.